app/src/main/java/com/example/mymovieinfo/ui/theme/Color.kt

```diff
@@ -5,3 +5,4 @@ import androidx.compose.ui.graphics.Color
 val Silver100 = Color(0xFFF0FFFF)
 //Figure out hex color values for various colors
 //create theme file to use text type, color, and  shape data
```

Comment:
Figure out hex color values for various colors
create theme file to use text type, color, and  shape data before next code review

Reply:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.